README.md

```diff
@@ -32,7 +32,7 @@ _Watch [this tutorial](https://elixircasts.io/deploying-with-gigalixir-%28revise
 _Note: The buildpack config are already included in this repo. Also, there is no need to provision a database._
 3. Create a new app.
 4. Set your AUTH_TOKEN from the Gigalixir console > configuration
-_Note: Your auth token will be used to authenitcate all API requests_
```

Comment:
i'll have you know I got this from the GW Bush dictionary. In the vein of "Nucular", etc. buuuuuuuuuut, I guess I'll go with the more traditional spelling.